pkg/server/httpAPI.go

```diff
@@ -521,7 +544,11 @@ func getInitialCluster(ctx context.Context, initialCluster string, etcdConn brty
 		logger.Warnf("Error with NewCluster() : %v", err)
 		return initialCluster
 	}
-	defer cli.Close()
+	defer func() {
+		if err1 := cli.Close(); err1 != nil {
```

Comment:
I used different error name to avoid clashes during function exit when defer is called, while an `err` is already defined and returned. The scope of the main function is used for the defer func as well, hence the possibility of a clash.

Reply:
Sure, makes sense. I'll make the change.